.github/workflows/build-and-push-image-build.yml

```
@@ -3,14 +3,41 @@ name: Build and push access-nri/build images
 on:
   workflow_dispatch:
     inputs:
+      spack-packages-build-required:
+        required: true
+        default: true
+        type: boolean
+        description: "A flag to note if building a new spack_packages image at the below version is required"
       spack-packages-version:
         required: true
         type: string
         default: "main"
         description: "Either a git tag or branch name for the ACCESS-NRI/spack_packages repository, which defaults to the main branch"
 
 jobs:
-  build-and-push-image:
+  base-spack-build-and-push-image:
+    if: inputs.spack-packages-build-required
+    uses: access-nri/workflows/.github/workflows/build-and-push-image.yml@main
```

```suggestion
    uses: access-nri/build-ci/.github/workflows/build-and-push-image.yml@main
```
This really confused me, but I think the `workflows` repo was renamed `build-ci`, so it resolves to `build-ci` but should probably be changed for clarity (assuming I am correct)

---

I can't suggest a change on unchanged text, but I'd make the same change at line 30 too.

---

Woah, that's good to know :D I thought it was some special auto-created repo that houses all of access-nri's workflows!